src/main/java/org/zalando/nakadi/service/subscription/state/StreamingState.java

```diff
@@ -260,6 +268,12 @@ public void onExit() {
                 kafkaConsumer = null;
             }
         }
+
+        getLog().debug("Unregister cursor reset listener");
```

Comment:
I think that it is too much to log this line (and if you think that it is better to log, then do it inside if)